### PR TITLE
Issue 40

### PR DESCRIPTION
**Pull request by KaraAJC**
Addressed the issue in the base repo to check whether social info links and images are not nil or empty before displaying it in the view.  Paired with Mike Chu(@mchu0010) on this issue

### CODE REVIEW

File: app/decorators/person_decorator.rb [hunk 1]
```diff
@@ -2,11 +2,11 @@ class PersonDecorator < Draper::Decorator
   delegate_all
 
   def twitter_url
-    "http://twitter.com/#{twitter}" if twitter
```
Comment (rachelober) [inline app/decorators/person_decorator.rb:5]: Hi @KaraAJC  can you change this pull request to use .blank? instead of `.nil? || .empty?`

Comment (rachelober) [inline app/decorators/person_decorator.rb:5]: http://apidock.com/rails/Object/blank%3F

Comment (rachelober) [inline app/decorators/person_decorator.rb:5]: Or change logic to be `.present?`

**Comment by KaraAJC** [comment]
@rachelober Chose `if .present?` based on this article: http://www.railstips.org/blog/archives/2008/12/01/unless-the-abused-ruby-conditional/

File: app/views/people/_person.html.haml [hunk 2]
```diff
@@ -1,17 +1,18 @@
 .speaker
   %a{ id: person.slug }
   .speaker__headshot
-    - if person.image_filename
```
Comment (rachelober) [inline app/views/people/_person.html.haml:4]: Hi Kara, thanks for fixing the other two files, this file also needs the updated convention. :-)

**Pull request merged**